test/bdd/features/search.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Searching for assets on the DKG
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+  Scenario: Publish and search assertion on the network with keywords
+    Given I setup 4 nodes
+    When I call publish on node 1 with validAssertion with keywords:
+      | keyword 1 | keyword 2 |
+    And I wait for last publish to finalize
+    And Last publish finished with status: COMPLETED
+    Given I call search request on node 1 with validAssertion for the keywords:
```

Comment:
We don't need this validAssertion in the step, we just need the number of the node and search keywords

Reply:
Maybe you can update it to be resultType so something like this:
I call search request on node 1 for result type assertion for the keywords: